resources/options_lookup.tsv

```diff
@@ -1668,7 +1668,7 @@ Cooking Range	"Electric, 100% Usage"	ResidentialApplianceCookingRange	fuel_type=
 Cooking Range	"Electric, 120% Usage"	ResidentialApplianceCookingRange	fuel_type=electric	cooktop_ef=0.74	oven_ef=0.11	has_elec_ignition=true	mult=1.2	location=auto																																							
 Cooking Range	"Electric, Induction"	ResidentialApplianceCookingRange	fuel_type=electric	cooktop_ef=0.84	oven_ef=0.11	has_elec_ignition=true	mult=1.0	location=auto																																							
 Cooking Range	None																																														
-Cooking Range Schedule	National Average	ResidentialApplianceCookingRange	"weekday_sch=0.007, 0.007, 0.004, 0.004, 0.007, 0.011, 0.025, 0.042, 0.046, 0.048, 0.042, 0.050, 0.057, 0.046, 0.057, 0.044, 0.092, 0.150, 0.117, 0.060, 0.035, 0.025, 0.016, 0.011"	"weekend_sch=0.007, 0.007, 0.004, 0.004, 0.007, 0.011, 0.025, 0.042, 0.046, 0.048, 0.042, 0.050, 0.057, 0.046, 0.057, 0.044, 0.092, 0.150, 0.117, 0.060, 0.035, 0.025, 0.016, 0.011"	"monthly_sch=1.097, 1.097, 0.991, 0.987, 0.991, 0.890, 0.896, 0.896, 0.890, 1.085, 1.085, 1.097"																																										
+Cooking Range Schedule	National Average	ResidentialApplianceCookingRange	"weekday_sch=0.01490, 0.0120, 0.0102, 0.0096, 0.0107, 0.0187, 0.0306, 0.0355, 0.0356, 0.0341, 0.0378, 0.0438, 0.0451, 0.0413, 0.0413, 0.0586, 0.0942, 0.1252, 0.1162, 0.0802, 0.0448, 0.0239, 0.0191, 0.0166"	"weekend_sch=0.01320, 0.0109, 0.0091, 0.0083, 0.0085, 0.0109, 0.0213, 0.0368, 0.0493, 0.0497, 0.0522, 0.0564, 0.0568, 0.0520, 0.0513, 0.0631, 0.0862, 0.1070, 0.0979, 0.0685, 0.0395, 0.0212, 0.0167, 0.0132"	"monthly_sch=1.097, 1.097, 0.991, 0.987, 0.991, 0.890, 0.896, 0.896, 0.890, 1.085, 1.085, 1.097"																																										

```

Comment:
Unintentional commit?

Reply:
This was in response to the Diversify cooking schedules analysis JIRA LP-548 (https://agile.nrel.gov/browse/LP-548). We decided to use schedules informed by the submeter data and the transferability study. At least until the stochastic occupant model is pulled into master.